src/wasm/literal.cpp

```diff
@@ -2352,41 +2352,41 @@ Literal Literal::narrowUToVecI16x8(const Literal& other) const {
 
 enum class LaneOrder { Low, High };
 
-template<size_t Lanes,
-         LaneArray<Lanes * 2> (Literal::*IntoLanes)() const,
-         LaneOrder Side>
+template<size_t Lanes, typename LaneFrom, typename LaneTo, LaneOrder Side>
 Literal extend(const Literal& vec) {
-  LaneArray<Lanes* 2> lanes = (vec.*IntoLanes)();
+  LaneArray<Lanes* 2> lanes = getLanes<LaneFrom, Lanes * 2>(vec);
```

Comment:
```suggestion
  LaneArray<Lanes * 2> lanes = getLanes<LaneFrom, Lanes * 2>(vec);
```

Reply:
Unfortunately formatter wants only `Lanes* 2` on the left side and only `Lanes * 2` on the right side 🤷

Reply:
Ugh it probably thinks it's a pointer. Oh well.